package-lock.json

```diff
@@ -0,0 +1,200 @@
+{
```

Comment:
This file also shouldn't exist at the top level and is a result of the top level package.json.  Also, we use yarn not NPM on this project. So package-lock files should never be created.